widgets/accordion/tpl/default.php

```diff
@@ -9,8 +9,16 @@
 ?>
 <div>
 	<div class="sow-accordion">
-	<?php foreach ( $panels as $panel ) : ?>
-		<div class="sow-accordion-panel<?php if ( $panel['initial_state'] == 'open' ) echo ' sow-accordion-panel-open'; ?>">
+	<?php foreach ( $panels as $i => $panel ) : ?>
+		<?php
```

Comment:
I feel like this code block is big enough that it should go in the widget's `get_template_vars`. Would just need to add `anchor` to the `$panels[$i]` array.

Reply:
Another thing to consider is adding filters more often. The anchor is a good example of something that a developer might want to filter.